homeronet/Plugin/YeahWoo.cs

```diff
@@ -0,0 +1,37 @@
+﻿using homeronet.Messages;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace homeronet.Plugin {
+
+    public class YeahWoo : IPlugin {
+        private List<string> _registeredCommands = new List<string>() { };
+
+        public void Startup() {
+        }
+
+        public void Shutdown() {
+        }
+
+        public Task<IStandardMessage> ProcessTextCommand(ITextCommand command) {
+            return null;
+        }
+
+        public List<string> RegisteredTextCommands {
+            get { return _registeredCommands; }
+        }
+
+        public Task<IStandardMessage> ProcessTextMessage(IStandardMessage message) {
+            return new Task<IStandardMessage>(() => {
+                if (message.Message == "yeah") {
```

Comment:
When core changes are in, circa 2018 we need to check if !message.SenderIsSelf && !message.SenderIsBot